website/docs/Pipelines/Optimus_Pipeline/starsolo-metrics.md

```diff
@@ -0,0 +1,104 @@
+# STAR Aligner Metrics
+The STAR aligner produces multiple text files containing library-level summary metrics, cell-level metrics, and UMI metrics. The Optimus workflow compresses these files into a single TAR. These outputs are directly from the aligner as different batches of the data are analyzed in parallel. 
+
+The STAR aligner metrics are supplemental to the [library-level metrics CSV](./Library-metrics.md) that is also produced by Optimus. Several of the calculations produced in the library metrics are directly based on the STAR aligner metrics. 
+
+The following sections describe these outputs. 
+
+## Align Features Metrics 
+The Align feature text file contains library-level metrics produced by the STARsolo alignment detailing the alignment of reads to genomic features during single-cell RNA-seq analysis. These metrics indicate how well reads map to specific genomic features or whether they failed to map due to various reasons. For example: 
+**noUnmapped** represents the number of reads that were not aligned to any feature in the genome.
+**noNoFeature** reflects reads that were aligned but did not map to any specific feature such as exons or genes.
+**MultiFeature** counts reads that were aligned to multiple features.
+**yesWLmatch** and **yesCellBarcodes** track how well reads match the barcode whitelist, an important step in identifying valid cell barcodes, which helps demultiplex the single-cell RNA-seq data​.
+
+Each of the table metrics gives insights into different stages of read alignment, from barcode matching to gene feature mapping, allowing you to assess the quality and accuracy of the alignment step in the pipeline.
+
+
+| Metrics name | Description | 
+| --- | --- | 
+| noUnmapped | Number of unmapped reads | 
+| noNoFeature | Number of reads not mapped to a feature. | 
+| MultiFeature | Number of reads aligned to multiple features. | 
+| subMultiFeatureMultiGenomic | Number of reads mapping to multiple genomic loci and multiple features. | 
+| noTooManyWLmatches | Number of reads not counted because their barcoded pair has too many matches to the whitelist. | 
+| noMMtoWLwithoutExact | Numbner of reads not counted because their barcoded pair has mismatches to the whitelist and there's no more reads supporting that barcode. | 
```

Comment:
```suggestion
| noMMtoWLwithoutExact | Number of reads not counted because their barcoded pair has mismatches to the whitelist and there's no more reads supporting that barcode. | 
```